packages/agoric-cli/src/commands/inter.js

```diff
@@ -0,0 +1,376 @@
+    },
+  } = quote;
+  const price = fmt.price({ numerator: amountOut, denominator: amountIn });
+
+  const amounts = objectMap(
+    { liquidatingCollateral, liquidatingDebt },
+    fmt.amount,
+  );
+  return { ...amounts, price };
+};
+
+/**
+ * Format amounts etc. in a bid OfferStatus
+ *
+ * @param {import('@agoric/smart-wallet/src/offers.js').OfferStatus &
+ *         { offerArgs: import('@agoric/inter-protocol/src/auction/auctionBook.js').BidSpec}} bid
+ * @param {import('agoric/src/lib/format.js').AssetDescriptor[]} assets
+ */
+export const fmtBid = (bid, assets) => {
+  const fmt = makeFormatters(assets);
+
+  const { offerArgs } = bid;
+  /** @type {{ price: string } | { discount: number }} */
+  const spec =
+    'offerPrice' in offerArgs
+      ? { price: fmt.price(offerArgs.offerPrice) }
+      : { discount: fmt.discount(offerArgs.offerBidScaling) };
+
+  const {
+    id,
+    error,
+    proposal: { give },
+    offerArgs: { want },
+    payouts,
+  } = bid;
+  const props = {
+    ...(give ? { give: fmt.record(give) } : {}),
+    ...(want ? { want: fmt.amount(want) } : {}),
+    ...(payouts ? { payouts: fmt.record(payouts) } : {}),
+    ...(error ? { error } : {}),
+  };
+  return harden({ id, ...spec, ...props });
+};
+
+/**
+ * @param {{
+ *   env: Partial<Record<string, string>>,
+ *   stdout: Pick<import('stream').Writable,'write'>,
+ *   stderr: Pick<import('stream').Writable,'write'>,
+ *   clock: () => number,
+ *   createCommand: // Note: includes access to process.stdout, .stderr, .exit
+ *     typeof import('commander').createCommand,
+ *   execFileSync: typeof import('child_process').execFileSync
+ * }} process
+ * @param {{ fetch: typeof window.fetch }} net
+ */
+export const makeInterCommand = async (
+  { env, stdout, stderr, clock, execFileSync, createCommand },
+  { fetch },
+) => {
+  const interCmd = createCommand('inter')
+    .description('Inter Protocol tool')
+    .option('--home [dir]', 'agd CosmosSDK application home directory')
+    .option(
+      '--keyring-backend [os|file|test]',
+      'keyring\'s backend (os|file|test) (default "os")',
+    );
+
+  const rpcTools = async () => {
+    const networkConfig = await getNetworkConfig(env).catch(err => {
+      throw Error(
+        `cannot get network config (${env.AGORIC_NET || 'local'}): ${
+          err.message
+        }`,
+      );
+    });
+    const { agoricNames, fromBoard, vstorage } = await makeRpcUtils(
+      { fetch },
+      networkConfig,
+    ).catch(err => {
+      throw new CommanderError(
+        1,
+        'RPC_FAIL',
+        `RPC failure (${env.AGORIC_NET || 'local'}): ${err.message}`,
+      );
+    });
+    const unserializer = boardSlottingMarshaller(fromBoard.convertSlotToVal);
+    return { networkConfig, agoricNames, fromBoard, unserializer, vstorage };
+  };
+
+  const bigintReplacer = (k, v) => (typeof v === 'bigint' ? `${v}` : v);
+  const pprint = v => JSON.stringify(v, bigintReplacer, 2);
```

Comment:
these are not specific to Inter at all. I'm surprised Endo doesn't export something for these. At last move to lib/format